lib/tasks/reset.rake

```diff
@@ -0,0 +1,30 @@
+namespace :hyrax do
+  namespace :reset do
+    desc 'Reset fedora / solr and corrisponding database tables w/o clearing other active record tables like users'
+    task works_and_collections: [:environment] do
+      confirm('You are about to delete all works and collections, this is not reversable!')
+      require 'active_fedora/cleaner'
+      ActiveFedora::Cleaner.clean!
+      Hyrax::PermissionTemplateAccess.delete_all
+      Hyrax::PermissionTemplate.delete_all
+      Delayed::Job.delete_all
```

Comment:
Delayed::Job is not a dependency of Hyrax.

Reply:
good point - @jpengst I forgot to take this out of the copy I sent you. Unfortunately there isn't a good generic way to clear all jobs in the queue, so we should remove this and we'll make a separate task that depends on this one in our various projects.